debug: true
bypass_behavior: false
planner_setup:
  via_points_speed_linear_ratio: 1.5
robot:
  wheelbase: 2.7
  min_turning_radius: 10.4
  max_vel_y: 0.0
  acc_lim_theta: 0.05
  acc_lim_x: 0.2
  max_vel_x_backwards: 0.01
  cmd_angle_instead_rotvel: true
teb_setup:
  homotopy_enabled: false
teb_trajectory:
  dt_ref: 1.0
  dt_hysteresis: 0.1
  min_obstacle_distance: 0.5
  inflation_distance: 0.6
  xy_goal_tolerance: 0.4
  yaw_goal_tolerance: 0.2
teb_weights:
  weight_obstacle: 40.0
  weight_viapoints: 4.0
  weight_optimaltime: 1.0
  weight_shortest_path: 0.0
  weight_kinematics_turning_radius: 10.0
  weight_kinematics_forward_drive: 5.0
  weight_kinematics_nh: 1000.0
  weight_max_vel_y: 0.0
